BeforeBuild.cmd

```diff
@@ -0,0 +1,32 @@
+@ECHO off
```

Comment:
Add a comment/remark at the top of the file stating this script configures the environment for the AppVeyor build.